### DISCUSSION TITLE
Horizontal UI!!

### DISCUSSION THREAD

**Discussion by crystian** [crystian/ComfyUI-Crystools#124] [Announcements]
finally! lots of changes, but is here!

I will work to improve it, there are some issues, but works

![image](https://github.com/user-attachments/assets/2e682747-0c5d-450a-8d2e-58abc9841f40)

**Comment by crystian** [comment]
now, you can configure the size!
![image](https://github.com/user-attachments/assets/11e74785-7bf5-4a92-8161-ad722da45468)

**Comment by yuart67** [comment]
please add Progress bar to horizontal menu otherwise there is no point in switching to new horizontal menu

**Comment by crystian** [comment]
they add a new progressbar, the mine does not need
